web/tests/example3.test.js

```diff
@@ -90,7 +91,7 @@ run_test("narrow_state", () => {
     const filter = new Filter(filter_terms);
 
     // And here is where we actually change state.
-    narrow_state.set_current_filter(filter);
+    override_rewire(narrow_state, "filter", () => filter);
```

Comment:
`override_rewire` isn't a pattern we should be using widely, and so we should keep it out of examples. I think this test could use `message_lists.update_current_message_list` or `message_lists.set_current`?

Reply:
Yeah, done.